README.md

```diff
@@ -1,17 +1,14 @@
 # OPA Gatekeeper Library
 
-This repository is a community-owned library of policies for the OPA Gatekeeper project.
+A community-owned library of policies for the OPA Gatekeeper project.
 
-## How to use the library
+## Usage
 
-The easiest way to apply a policy from this library is to download and apply the `constraint.yaml` and `template.yaml` provided in each directory
+Apply the `constraint.yaml` and `template.yaml` provided in each directory under `library/`
 
-For example
-
-    cd library/general/httpsonly/
-    kubectl apply -f constraint.yaml
-    kubectl apply -f template.yaml
-
-## How to contribute to the library
-
-TODO.
+```bash
+cd library/general/httpsonly/
+kubectl apply -f template.yaml
+kubectl apply -f samples/ingress-https-only/constraint.yaml
+kubectl apply -f library/general/httpsonly/sync.yaml
```

Comment:
I believe this example doesn't need sync directly unless GK is running with OPA cache. Can we add a note here that this is optional?

Reply:
done